pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -294,6 +294,7 @@ func (w *nfdTopologyUpdater) updateNodeResourceTopology(zoneInfo v1alpha2.ZoneLi
 		nrtNew := v1alpha2.NodeResourceTopology{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:            w.nodeName,
+				Labels:          map[string]string{"owner-pod": os.Getenv("POD_NAME")},
```

Comment:
Let's use https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L295 that is the to-go way to define ownership instead of a custom label

Reply:
My bad as @PiotrProkop commented to my offline, you can not set OwnerReference to a Cluster scoped object